samples/nrf9160/memfault/README.rst

```diff
@@ -193,7 +193,19 @@ Before testing, ensure that your device is configured with your Memfault project
         <dbg> <mflt>: Body: Accepted
         <dbg> <mflt>: No more data to send
 
-#. Press <TAB> on your keyboard to confirm that the Memfault shell is working.
+#. Upload the symbol file generated from your build to your Memfault account so that the information from your application can be parsed. The symbol file is located in the build folder: :file:`memfault/build/zephyr/zephyr.elf`:
+
+    a. In a web browser, navigate to `Memfault`_.
+
+    b. Login to your account and select the project you created earlier.
+
+    c. Navigate to :guilabel:`Fleet` > :guilabel:`Devices` in the left menu.
+
+        You can see your newly connected device and the software version in the list.
+
+    d. Click on the software version number for your device and then the :guilabel:`Upload` button to upload the symbol file.
```

Comment:
```suggestion
#. Upload the symbol file generated from your build to your Memfault account so that the information from your application can be parsed. The symbol file is located in the build folder: :file:`memfault/build/zephyr/zephyr.elf`:

   a. In a web browser, navigate to `Memfault`_.
   b. Login to your account and select the project you created earlier.
   c. Navigate to :guilabel:`Fleet` > :guilabel:`Devices` in the left menu.

      You can see your newly connected device and the software version in the list.

   d. Click on the software version number for your device and then the :guilabel:`Upload` button to upload the symbol file.
```